tools/Makefile

```diff
@@ -6,8 +6,13 @@ PYTHON :=
 PYTHON_VERSION := 3.7.3
 CUPY_VERSION := 6.0.0
 CUDA_VERSION := 10.0
-# PyTorch version: 0.4.1 or 1.0.0 or 1.0.1
+# PyTorch version: 0.4.1, 1.0.0, 1.0.1, 1.1.0, 1.2.0, 1.3.0, 1.3.1
```

Comment:
1.4 is not working?

Reply:
I did not test the version but should be working if there is no big difference from 1.3.1

Reply:
@Emrys365 already tested and it's working (see https://github.com/espnet/espnet/pull/1527)